conf/deployment/vsphere/upi_1az_rhcos_lowerreq_vmfs_3m_3w.yaml

```diff
@@ -0,0 +1,16 @@
+---
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+ENV_DATA:
+  platform: 'vsphere'
+  deployment_type: 'upi'
+  worker_replicas: 3
+  master_replicas: 3
+  worker_num_cpus: '8'
+  master_num_cpus: '8'
+  master_memory: '32768'
```

Comment:
are we planning to increase master memory  because in vsphere config we use 16GB memory https://github.com/red-hat-storage/ocs-ci/blob/master/conf/deployment/vsphere/upi_1az_rhcos_vmfs_3m_3w.yaml

Reply:
Correct, will change that